shoulder_pan_joint: 0.0
shoulder_lift_joint: -1.57     # dobra o ombro para cima
elbow_joint: 1.57              # dobra o cotovelo para cima
wrist_1_joint: 0.0             # mant√©m o punho alinhado
wrist_2_joint: 0.0
wrist_3_joint: 0.0

